### PR TITLE
Doc: Rename "Event Dependent Configuration" Chapter.

### PR DESCRIPTION
**Pull request by 7lima**
The current chapter title does not reflect the contents so it's hard to find it again later on when you're looking for info about accessing fields.

### CODE REVIEW

File: docs/static/configuration.asciidoc [hunk 1]
```diff
@@ -278,7 +278,7 @@ input { # comments can appear at the end of a line, too
 ----------------------------------
 
 [[event-dependent-configuration]]
-=== Event Dependent Configuration
+=== Accessing Event Data and Fields
```
Comment (suyograo) [inline docs/static/configuration.asciidoc]: how about `Accessing Event Data and Fields In Configuration`? IMO Configuration is key here..

**Comment by suyograo** [comment]
@7lima can you update this?

**Comment by 7lima** [comment]
@suyograo If you think the Configuration keyword from the superordinate "Configuring Logstash" chapter is necessary in the section's title then I would choose _"Accessing Event Data and Fields in the Configuration"_ like in the following "Using Environment Variables in the Configuration" section.

**Comment by suyograo** [comment]
@7lima +1 on "Accessing Event Data and Fields in the Configuration"

**Comment by dedemorton** [comment]
This PR was never completed, so I'm replacing it with #6767

**Pull request closed without merging**